### ISSUE TITLE
Need a way to indicate that an element is a JSON object with arbitrary properties

### ISSUE THREAD

**Issue by lmckenzi** [FHIR/fhir-tools-ig#6]
In CRD, there's an extension (https://build.fhir.org/ig/HL7/davinci-crd/deviations.html#hook-configuration-extension) that allows arbitrary children.  (What's allowed is actually determined by a different extension returned in a different call.)  I need a way to declare a type to essentially be "JSONObject" and have that work, or something similar.

**Comment by grahamegrieve** [comment]
I thought this is already defined here: https://build.fhir.org/ig/FHIR/fhir-tools-ig/StructureDefinition-CDSHooksExtension-DaVinci-configoptions.html

**Comment by lmckenzi** [comment]
No.  That one has specific named children.  There's a companion extension that appears on the hook rather than the service look-up.  The service look-up extension has specific named properties.  The one on the hook invocation itself is open-ended.  It can contain any property you like - specifically anything that the payer listed as a candidate in the values they listed in the configuration hook.  So I need a different capability - one where I can say "this can contain anything" rather than listing specific content.

I also need a way for the 'root' of a CDS hook extension to just be a simple type - no children.

**Comment by grahamegrieve** [comment]
"just anything" doesn't sound very useful, really. Why both describing it?

**Comment by lmckenzi** [comment]
From a validation perspective, I need to be able to say: "This element, with this name, can appear 'here', and beneath it can be anything".
